build/BUILD

```diff
@@ -0,0 +1,41 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_k8s_repo_infra//defs:build.bzl", "release_filegroup")
+load("@io_k8s_repo_infra//defs:pkg.bzl", "pkg_tar")
+
+pkg_tar(
+    name = "kubernetes-server-linux-amd64",
+    srcs = [
```

Comment:
are you basically relying on the implementation detail that pkg_tar won't overwrite an existing file? that seems a little fragile and worth commenting at the very least

Reply:
Yes, :). Added a comment.